custodia/secrets.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, config, section):
 
     def _db_key(self, trail):
         if len(trail) < 2:
-            self.logger.debug(
+            self.logger.exception(
```

Comment:
```logger.debug()``` was correct, sorry for not being clear here.

```logger.exception()``` makes only sense in the except block of a try/except statement.